src/data/data.yaml

```
@@ -0,0 +1,43 @@
+name: Data @ Mozilla
+summary: Data Science and Data Engineering. Helping Mozillians make better, data-informed decisions.
+icon: database
+introduction: |
+  ## About Data @ Mozilla
+
+  The Data group's vision is a sustainable, global and independent Mozilla that ethically leverages data for the common good.
+  Data @ Mozilla is composed of two sister teams:
+
+  * Data Engineering: Works on the data platform that allows us to gather and analyze data about our products. This includes
+    the [Glean SDK](https://mozilla.github.io/glean) as well as 
+    [our data ingestion pipeline](https://docs.telemetry.mozilla.org/concepts/pipeline/gcp_data_pipeline.html).
+  * Data Science: Helps the rest of Mozilla make data informed decisions using the data platform, applying a broad range of
+    statistical and analytical skills.
+
+  A good starting point for learning more is our [team wiki page](https://wiki.mozilla.org/Data).
+
+  ## How Do I Get Started?
+
+  If you are working on a bug (in Bugzilla), you may need to [create a new account](https://bugzilla.mozilla.org/createaccount.cgi).
+  If you are working on an issue (in Github), you will need a Github account.
+
+  Comment in the bug or issue to say that you are working on it, and ask any questions that you have at that time.
+  But do your research!
+  Look the other comments, look at the documentation and source code, and try to figure out as much as you can first.
+  This helps you learn more about the data platform and understand better the bug you're fixing or feature you're adding.
+
+  ## How Do I Get Help?
+
+  The best place to talk about a bug or issue is in the comments.
+  Don't be afraid to ask questions or describe how you are solving the problem.
+  That way, anyone watching the bug can answer your questions or offer useful advice.
+
+  Individual data projects may have project-specific channels or forums for asking questions and getting advice.
+  We are also available on [matrix.io](https://chat.mozilla.org/#/room/#telemetry:mozilla.org) in the `#telemetry` channel in the mozilla.org community.
+
+products:
+  - "Data Platform and Tools"
+
+repositories:
+  - mozilla/glean: "good first issue"
+  - mozilla/glean.js: "good first issue"
+  - mozilla/glean-dictionary: "good first issue"
```

If we want this to supersede "Telemetry Dashboards", we should add the relevant sections here:

```suggestion
  - mozilla/glean-dictionary: "good first issue"
  - mozilla/telemetry-dashboard: ['mentored', 'good first issue']
```

---

I'm happy with this superseding "Telemetry Dashboards"

@djmitche I notice that there's a ghost "project" for Telemetry as well (`Toolkit::Telemetry` on BMO, part of Firefox Desktop). Should we umbrella that here, too? Or would that be a little odd.

---

I'm just the person responsible for clicking merge here -- y'all know the projects better than me :D

---

nit: Both the Glean SDK and Glean.js actually use a [good first **bug**] tag. Not sure that makes a difference here. Also, when we start getting a bit of traction on Glean.js I am thinking about having a [help wanted] tag for bugs that might not be great for first constributions, but welcome contributor help. I guess it would be no harm to already include that tag here.

---

@brizental oops, good catch! We actually want to index the bugzilla components for glean + glean.js, not github. I'll fix this. It seems like codetribute has its own heuristics for finding good first bugs and mentored bugs based on the bugzilla conventions, so long as you align with those I think you should be good:

https://github.com/mozilla-frontend-infra/codetribute/blob/main/src/views/Languages/index.jsx#L98